_tests/browser_tests/.eslintrc

```diff
@@ -0,0 +1,10 @@
+env:
+  node: true
```

Comment:
This shouldn't be needed. It's already true [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/.eslintrc#L30).